spec-0007/index.md

```diff
@@ -129,12 +120,37 @@ This is implemented using:
    It changes the documentation and auto-completion to only advertise the new `rng` parameter.
 
 ```python
-_NoValue = object()  # singleton to indicate not explicitly passed
+import numpy as np
+import functools
+import numbers
+import warnings
 
+_NoValue = object()  # singleton to indicate not explicitly passed
 
-def check_random_state(seed=_NoValue, rng=_NoValue):
+# Should this specify the version number in which the behavior will change?
```

Comment:
```suggestion
```
Should the warning specify the version number in which the behavior will change?

Reply:
Let's address this in the main PR after this PR is merged. But in SciPy, if we emit a `DeprecationWarning` or `FutureWarning`, it now specifies the version in which the behavior will change. The clear timeline is not just important to users, but also developers. (In the past, ambiguous deprecations often were left as warnings for several years.)

Reply:
Added version number.